src/status_im/contexts/wallet/signals.cljs

```diff
@@ -2,28 +2,29 @@
   (:require
     [oops.core :as oops]
     [status-im.constants :as constants]
+    [status-im.contexts.wallet.db :as db]
     [taoensso.timbre :as log]
     [utils.re-frame :as rf]
     [utils.transforms :as transforms]))
 
 (rf/reg-event-fx
  :wallet/pending-transaction-status-changed-received
  (fn [{:keys [db]} [{:keys [message]}]]
-   (let [details                      (transforms/json->clj message)
-         tx-hash                      (:hash details)
-         tx-status                    (:status details)
-         status                       (cond
-                                        (= tx-status constants/transaction-status-success)
-                                        :confirmed
-                                        (= tx-status constants/transaction-status-pending)
-                                        :pending
-                                        (= tx-status constants/transaction-status-failed)
-                                        :failed)
-         swap-approval-transaction-id (get-in db [:wallet :ui :swap :approval-transaction-id])
-         swap-approval-transaction?   (= swap-approval-transaction-id tx-hash)
-         swap-transaction-ids         (get-in db [:wallet :swap-transaction-ids])
-         swap-transaction?            (and swap-transaction-ids
-                                           (contains? swap-transaction-ids tx-hash))]
+   (let [details                                (transforms/json->clj message)
+         tx-hash                                (:hash details)
+         tx-status                              (:status details)
+         status                                 (cond
+                                                  (= tx-status constants/transaction-status-success)
+                                                  :confirmed
+                                                  (= tx-status constants/transaction-status-pending)
+                                                  :pending
+                                                  (= tx-status constants/transaction-status-failed)
+                                                  :failed)
+         {:keys [swap-approval-transaction-id]} (get-in db db/swap)
```

Comment:
wondering if `(conj db/swap :swap-approval-transaction-id)` is better since we avoid a long let binding alignment. wdyt?

Reply:
It would be longer actually:
```clojure
```

Reply:
But it won't force all bindings to be displaced too far.

It's fine, just I don't like (as others, I guess) when all the code is significantly moved to the right